internal/git/listrefs.go

```diff
@@ -0,0 +1,52 @@
+package git
+
+import (
+	"emperror.dev/errors"
+	"strings"
+)
+
+func (r *Repo) ListRefs(showRef *ListRefs) ([]RefInfo, error) {
```

Comment:
also annoying to have the type-struct same as method name. hard to read.

Reply:
Eh it's the best we can do given the lack of support for keyword arguments. The idea is to add ability to specify more fields in the future if necessary. It might not matter a whole lot for this method, but I wanted to be consistent with the `git` API.